sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -383,19 +385,15 @@ class ProjectOverviewView extends VerticalLayout {
      * @param sampleCount The total number of samples registered
      */
     private static State determineCompleteness(SampleCount sampleCount) {
-        if (sampleCount.failingSamples > 0){
+        if (sampleCount.failingSamples > 0) {
```

Comment:
I do not know whether I mentioned it before but I like that you improved the formatting here.

Reply:
Thanks to you code sensei 🧠